lib/segment/src/index/field_index/full_text_index/immutable_text_index.rs

```diff
@@ -67,10 +66,8 @@ impl ImmutableFullTextIndex {
 
     /// Open and load immutable full text index from mmap storage
     pub fn open_mmap(index: MmapFullTextIndex) -> Self {
-        let inverted_index = match index.inverted_index.storage.postings {
-            MmapPostingsEnum::Ids(_) => ImmutableInvertedIndex::ids_empty(),
-            MmapPostingsEnum::WithPositions(_) => ImmutableInvertedIndex::positions_empty(),
-        };
+        let inverted_index = ImmutableInvertedIndex::from(&index.inverted_index);
+
```

Comment:
🤦

Reply:
I missed this line here: https://github.com/qdrant/qdrant/pull/7004/files#diff-11f65a2be5444195ffec029c4e72c07a71135e9050d228d00cdb9044f3518d06L138

☹️